base/settings/R/check.all.settings.R

```diff
@@ -398,7 +398,7 @@ check.settings <- function(settings, force = FALSE) {
         "qsub not specified using default value :", settings$host$qsub)
     }
     if (is.null(settings$host$qsub.jobid)) {
-      settings$host$qsub.jobid <- "Your job ([0-9]+) .*"
+      settings$host$qsub.jobid <- "Your job ^[A-Za-z0-9_-]*$"
```

Comment:
Not sure this will catch many Job IDs.  For example, if the HPC message for a submitted SLURM script is "Job 190231 submitted" or "Submitted as job 19203" this regex wouldn't work I think.  The idea in #2959 was to make the default regex able to catch any consecutive group of numbers anywhere in whatever the submission message is.

Reply:
oh sorry I forgot about adding blank space in the regex, going to fix it now

Reply:
So it's not the blank space, it's that my HPC doesn't print "Your job", so this regex doesn't work. 

Try creating a few test submission messages like "Your job 812973 has been submitted", "Job 109238 submitted", "Submitted 1282", etc. and then using `sub()` to figure out what regex pattern returns only the numeric part.

I was pretty sure `"\\D*([0-9]+)\\D*"` would work, but when I tried it in an actual pecan.xml file it didn't.  I'm not sure why.  My HPC prints "Submitted batch job 4094596" upon SLURM script submission.  The regex that works for me is `".* ([0-9]+)"`

Reply:
you can change this by setting it in your pecan.xml:
```
<host>
  ...
  <qsub.jobid>.* ([0-9]+).*</qsub.jobid>
</host>
```
This is a very generic one, and might end up eating to much, or not mach. You can check it at: https://regex101.com/